uma-rs-resteasy/src/main/java/org/gluu/oxd/rs/protect/resteasy/ResourceRegistrar.java

```diff
@@ -92,12 +92,13 @@ private void register(RsResource rsResource) {
                     resource.setScopes(condition.getScopes());
                 }
                 //set creation and expiration timestamp
-                if (safeToInt(rsResource.getIat()) && safeToInt(rsResource.getExp())) {
-                    if(rsResource.getExp() > rsResource.getIat()){
+                if (isSafeToInt(rsResource.getIat()) && isSafeToInt(rsResource.getExp())) {
```

Comment:
if `iat` is null (not set) how would it go to line 101 ?

Reply:
Done. As date check already present in AS, so we removed `if(rsResource.getExp() > rsResource.getIat())` condition.